Description: install missing attributes.h header file
Author: Agathe Porte <agathe.porte@oss.qualcomm.com>

The `pcm.h` file includes the `attributes.h` header file, which is not
installed on the system. This results in an include error when trying to build
programs that depend on libtinyalsa.

Install the missing header so that including `pcm.h` does not result in an
import error.

--- a/include/tinyalsa/meson.build
+++ b/include/tinyalsa/meson.build
@@ -1,5 +1,6 @@
 tinyalsa_headers = [
   'asoundlib.h',
+  'attributes.h',
   'interval.h',
   'limits.h',
   'mixer.h',
